app/src/components/Map/SettingsPanel.jsx

```diff
@@ -1,34 +1,60 @@
 import React, { Component } from 'react';
 import SettingsPanelStyle from '../../../styles/components/c-settings_panel.scss';
+import { DEFAULT_VESSEL_COLOR } from '../../constants';
 
 class SettingsPanel extends Component {
 
   render() {
+    const colors = [
+      DEFAULT_VESSEL_COLOR,
+      '#e41ccc',
+      '#fe5f07',
+      '#a73c8a',
+      '#9e7854',
+      '#61c200',
+      '#c7ad23'
```

Comment:
these are temporary. for the final thing, we either remove the color changing option altogether, or add a proper color picker